ga4gh/frontend.py

```diff
@@ -261,6 +300,26 @@ def assertCorrectVersion(version):
         raise exceptions.VersionNotSupportedException()
 
 
+def authenticated(handler):
+    @functools.wraps(handler)
+    def checkAuthentication(*args, **kwargs):
+        """
+        The request should come with an Authorization header of form:
+        Authorization: Bearer <token>
+        We are interested in the token.
+
+        If there is no oidcClient, we are running naked and we don't check.
+        """
+        if app.oidcClient:
+            requestToken = request.headers.get('Authorization')
+            requestToken = requestToken.split()[1] if requestToken else None
+            if not app.tokenMap.get(requestToken):
```

Comment:
Here too, actually: presumably `app.tokenMap.get(requestToken)` returns None if the key isn't defined. Sorry to bang on about this, but I feel strongly about this as a readability issue, and the potential for unexpected side effects when converting to boolean makes me very nervous, particularly and especially in such a critical function.

Here's the PEP8 quote:

> Also, beware of writing if x when you really mean if x is not None -- e.g. when testing whether a variable or argument that defaults to None was set to some other value. The other value might have a type (such as a container) that could be false in a boolean context!